### PR TITLE
Switched WorkflowRegistry to use ValueTuples to improve code readability 

### PR DESCRIPTION
**Pull request by DevsAnon**

### CODE REVIEW

File: src/WorkflowCore/Services/WorkflowRegistry.cs [hunk 1]
```diff
@@ -11,7 +11,7 @@ namespace WorkflowCore.Services
     public class WorkflowRegistry : IWorkflowRegistry
     {
         private readonly IServiceProvider _serviceProvider;
-        private readonly BlockingCollection<Tuple<string, int, WorkflowDefinition>> _registry = new BlockingCollection<Tuple<string, int, WorkflowDefinition>>();
+        private readonly BlockingCollection<(string, int, WorkflowDefinition)> _registry = new BlockingCollection<(string, int, WorkflowDefinition)>();
```
Comment (danielgerlag) [inline src/WorkflowCore/Services/WorkflowRegistry.cs:14]: could we use named tuples?

**Review by danielgerlag — APPROVED** [review]

File: src/WorkflowCore/Services/WorkflowRegistry.cs (hunk 1 above)
Comment (DevsAnon) [reply src/WorkflowCore/Services/WorkflowRegistry.cs]: Done

File: src/WorkflowCore/Services/WorkflowRegistry.cs [hunk 2]
```diff
@@ -11,7 +11,7 @@ namespace WorkflowCore.Services
     public class WorkflowRegistry : IWorkflowRegistry
     {
         private readonly IServiceProvider _serviceProvider;
-        private readonly BlockingCollection<Tuple<string, int, WorkflowDefinition>> _registry = new BlockingCollection<Tuple<string, int, WorkflowDefinition>>();
+        private readonly BlockingCollection<(string workfloId, int version, WorkflowDefinition definition)> _registry = new BlockingCollection<(string, int, WorkflowDefinition)>();
```
Comment (danielgerlag) [inline src/WorkflowCore/Services/WorkflowRegistry.cs:14]: typo in `workfloID`

File: src/WorkflowCore/Services/WorkflowRegistry.cs [hunk 3]
```diff
@@ -20,24 +20,24 @@ public WorkflowRegistry(IServiceProvider serviceProvider)
 
         public WorkflowDefinition GetDefinition(string workflowId, int? version = null)
         {
+            (string workflowId, int version, WorkflowDefinition definition) workflowEntry;
             if (version.HasValue)
             {
-                var entry = _registry.FirstOrDefault(x => x.Item1 == workflowId && x.Item2 == version.Value);
-                // TODO: What in the heck does Item3 mean?
-                return entry?.Item3;
+                workflowEntry = _registry.FirstOrDefault(x => x.Item1 == workflowId && x.Item2 == version.Value);
```
Comment (danielgerlag) [inline src/WorkflowCore/Services/WorkflowRegistry.cs:26]: can we use the names in the queries for readability?

File: src/WorkflowCore/Services/WorkflowRegistry.cs (hunk 2 above)
Comment (DevsAnon) [reply src/WorkflowCore/Services/WorkflowRegistry.cs]: Done

**Comment by DevsAnon** [comment]
Current state? Ready to merge?

**Comment by DevsAnon** [comment]
@danielgerlag Will this be merged?

**Pull request merged**